packages/gatsby-plugin-google-gtag/README.md

```diff
@@ -46,6 +46,8 @@ module.exports = {
           respectDNT: true,
           // Avoids sending pageview hits from custom paths
           exclude: ["/preview/**", "/do-not-track/me/too/"],
+          // Defaults to https://www.googletagmanager.com
+          selfHostedOrigin: "YOUR_SELF_HOSTED_ORIGIN",
```

Comment:
Very minor request, could we update this to just be "origin"? Perhaps there are some use case for a third-party handling analytics, rather than something self hosted. In that case, the configuration might not be read clearly.

Reply:
I got inspired by the https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-google-tagmanager it is using the same variable for exactly the same functionality.